source/data/build.R

```diff
@@ -1,7 +1,10 @@
-args = commandArgs(TRUE)
+library(yaml)
+PATHS <- yaml.load_file('constants.yaml')
+args  <- commandArgs(TRUE)
+
 main <- function(i){
     x    <- seq(-10, 10, 0.1)
-    write.table(x, sprintf("build/data/%s.txt", i), sep = "|", 
+    write.table(x, sprintf("%s/%s.txt", PATHS$build$data, i), sep = "|", 
```

Comment:
Can probably make a note with the previous comment on the outline to rethink this later if we want.